content/bash-scripting-basics.md

```diff
@@ -0,0 +1,158 @@
+
+```bash
+read firstName
+```
+
+If we were to run our script in our Terminal, it will first output a line that asks "What is your first name?". It will then sit and wait for the user to input text. Once the user inputs text and presses Enter, the script will end. We have not given the script any commands to use this information that the user has input. Let's add some output that shows what the user has typed in their Terminal. Add a new line to your script:
+
+```bash
+echo "Your first name is: $firstName"
+```
+
+Notice that in order to use our `firstName` variable, we use the `$` in front of it. This will tell our terminal that we intend to use the variable called `firstName`. If I were to run the script, it will ask for my first name. I will input the name "Garrett", and the script will then output the following line:
+`Your first name is: Garrett`
+
+Now, let's add another question to this script. This time, we will ask for the users favorite color. We will then use the `read` command again to get user input and store it in a variable called `favoriteColor` and then finally output a sentence that shows what the user input:
+
+```bash
+echo "What is your favorite color?"
+read favoriteColor
+echo "Your favorite color is: $favoriteColor"
+```
+
+At this point, you should have a working script that asks the user for their name and their favorite color and then reacts to this information by outputting a sentence that reflects this information. The full script at this point should look like this:
+
+```bash
+#!/usr/bin/env bash
+echo "What is your first name?"
+read firstName
+echo "Your first name is: $firstName"
+echo "What is your favorite color?"
+read favoriteColor
+echo "Your favorite color is: $favoriteColor"
+```
+
+### If-Then Statements
+
+Sometimes we want different things to happen depending on what inputs/variables are stored. For instance: if a file exists, then do *a*, but if the file does not exist, then do *b*. Or if a number is higher than *x* then do *a*, but if the number is lower than *x*, then do *b*. We can do this with an `if` statement.
+
+For this, we will be asking the user to input a number. If the number is higher than *x* we will do *a*, but if the number is lower than *x*, we will do *b*.
+
+Add a new line to your script. We will be making the target number 23, but you can pick any number you want. We will be putting the target number into a variable called `targetNumber`. Add the following line to your script:
+
+```bash
+targetNumber=23
+```
+
+We will now ask the user to input a number, and store that number to a variable called `guess`. Add the following lines to your script:
+
+```bash
+echo "Guess the number:"
+read guess
+```
+
+We now need to create and `if` statement to compare the `targetNumber` to the user input `guess` variable. The script will output different sentences depending on if the user is too low, too high, or if they guess correctly. Add the following lines to your script:
```

Comment:
> different sentences depending on if the user is too low

"depending on whether" would look better than "depending on if". There is another occurrence in the sentence below the code block below.

Reply:
This was not addressed